index.js

```diff
@@ -62,18 +62,26 @@ function parse_desc_stream(descstream, callback) {
     descstream.on('end', finisher);
 }
 
+function Dependency(package, version) {
+  this.package = package;
+  if(version)
+    this.version = version;
```

Comment:
So we don't want to separate the `>=`? FWIW in theory it can also be `==` or `<=` although that's hardly ever used.

Reply:
The parser leaves the `>-` part in the version. So the version already contains e.g  "> 3.0"

Reply:
I am asking if we should separate those or not.

Reply:
Ah ok. I don't think so, I don't see how the `>=` by itself is meaingful without the number.

Reply:
Well, if you want to check if a version requirement holds, then you need to separate them every time, so we might already in the parser. Basically inside the parens it is a tiny DSL, so we might as well parse it.